src/components/analyses/toolbar/AnalysesDotMenu.js

```diff
@@ -184,26 +202,25 @@ function DotMenuItems(props) {
     ];
 }
 
-function AnalysesDotMenu(props) {
-    const {
-        baseId,
-        ButtonProps,
-        username,
-        selected,
-        getSelectedAnalyses,
-    } = props;
-
-    const isSingleSelection = selected?.length === 1;
+function AnalysesDotMenu({
+    ButtonProps,
+    username,
+    getSelectedAnalyses,
+    ...props
+}) {
+    const { baseId, isSingleSelection } = props;
```

Comment:
These `baseId` and `isSingleSelection` props need to be spread down into the `DotMenuItems` component below, but the props above do not.

This was also an attempt at removing the `Function components cannot be given refs` warning, but now I'm not sure it's related to the `AnalysesDotMenu` props 🤔